src/plugins/console/server/routes/api/console/autocomplete_entities/register_get_route.ts

```diff
@@ -70,16 +71,26 @@ export function registerGetRoute({ router, lib: { handleEsError } }: RouteDepend
           });
         }
 
-        const esClient = (await ctx.core).elasticsearch.client;
         const mappings = await getMappings(esClient, settings);
+        const maxMappingSizeExceeded = Object.keys(mappings).length > 10 * 1024 * 1024;
```

Comment:
@cjcenizal  @sabarasaba I'm not sure that we should proceed with that changes. I see we are still overloading our ES instance, can we just limit the size of the data returned with displaying the warning message for that case?

Probably we can specify the next option with setting it to e.g. 1000
`maxResponseSize?: number;`